lib/loga/log_subscribers/action_mailer.rb

```diff
@@ -0,0 +1,28 @@
+module Loga
+  module LogSubscribers
+    class ActionMailer < ActiveSupport::LogSubscriber
+      def deliver(event)
+        emails   = event.payload[:to].join(',')
+        mailer   = event.payload[:mailer]
+        duration = event.duration
+
+        message = "action_mailer.delivery to #{emails} via #{mailer} in #{duration}ms"
+        log_message = Loga.formatter.call('INFO', event.time, nil, message)
```

Comment:
I think it's important to incorporate the feedback I posted in this [comment](https://github.com/FundingCircle/loga/pull/106#discussion_r182988286).

The formatter isn't meant to be accessed directly. Instead we should pass in `Loga::Event` with the necessary information which will be picked up by the formatter.